project/scripts/cmdScaladocTests

```diff
@@ -40,6 +40,6 @@ dist/target/pack/bin/scaladoc \
   -project-footer "Copyright (c) 2002-2022, LAMP/EPFL" \
   -default-template static-site-main \
   -author -groups -revision main -project-version "${DOTTY_BOOTSTRAPPED_VERSION}" \
-  "-quick-links:Learn::https://docs.scala-lang.org/,Install::https://www.scala-lang.org/download/,Playground::https://scastie.scala-lang.org,Find A Library::https://index.scala-lang.org,Community::https://www.scala-lang.org/community/,Blog::https://www.scala-lang.org/blog/" \
+  "-quick-links:Learn::https://docs.scala-lang.org/,Install::https://www.scala-lang.org/download/,Playground::https://scastie.scala-lang.org,Find A Library::https://index.scala-lang.org,Community::https://www.scala-lang.org/community/,Blog::https://www.scala-lang.org/blog/" \
```

Comment:
What did you change here?

Reply:
This changed normal spaces to non-breaking spaces to have the same behavior in the script tests and sbt tests.

Reply:
https://github.com/lampepfl/dotty/pull/16306/commits/37c2b9cdbda7df2dffd5ee8374b6bb6f2ac6d6cd is a cherry-picked commit from https://github.com/lampepfl/dotty/pull/16310 that is doing even more to finally fix the tests.